packages/gatsby-source-wordpress/src/steps/ingest-remote-schema/identify-and-store-ingestable-types.js

```diff
@@ -46,6 +50,23 @@ const identifyAndStoreIngestableFieldsAndTypes = async () => {
       continue
     }
 
+    if (!interfaceType.fields) {
+      continue
+    }
+
+    const typesThatImplementInterface =
+      getTypesThatImplementInterfaceType(interfaceType)
+
+    const shouldSkipInterfaceType = !transformFields({
+      fields: interfaceType.fields,
+      parentType: interfaceType,
+      parentInterfacesImplementingTypes: typesThatImplementInterface,
+    })
+
+    if (shouldSkipInterfaceType && interfaceType.name !== `Node`) {
```

Comment:
We always need the `WpNode` type no matter what and it causes other problems if it's auto-excluded